aws/table_aws_iam_user_ssh_key.go

```diff
@@ -0,0 +1,214 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsIamUserSshKey(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_iam_user_ssh_key",
+		Description: "AWS IAM SSH public key.",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"ssh_public_key_id", "user_name"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ValidationError", "NoSuchEntity", "InvalidParameter"}),
+			},
+			Hydrate: getIamSshPublicKey,
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listIamUsers,
+			Hydrate:       listIamSshPublicKeys,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "user_name", Require: plugin.Optional},
+			},
+		},
+		Columns: awsGlobalRegionColumns([]*plugin.Column{
+			{
+				Name:        "ssh_public_key_id",
+				Description: "The unique identifier for the SSH public key.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SSHPublicKeyId"),
+			},
+			{
+				Name:        "ssh_public_key_body_pem",
```

Comment:
This column is one I'm unsure about, should we show it as this format?

Reply:
@cbruno10 The API [GetSSHPublicKey](https://docs.aws.amazon.com/IAM/latest/APIReference/API_GetSSHPublicKey.html) provides the public key body with both format(`SSH` and `PEM`). From this API there is property called `SSHPublicKeyBody` which provides the key body by providing the `Encoding` value(`SSH` or `PEM`) in the API parameter. So we have added both of the column (`ssh_public_key_body_pem` and `ssh_public_key_body_rsa`) in this table.

Reply:
@cbruno10 We may have similar scenarios where we can upload the same SSH public key in different formats to enable my access i.e. I can query the column and get the Public key > provide the same in other application.
![image](https://user-images.githubusercontent.com/47887552/229435887-37490437-aff9-477e-a957-15bc37141e1a.png)